### ISSUE TITLE
Name Property for 'Automatic check for updates' Combo box in 'preferences' window is not defined 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#351]
### Steps to Reproduce
1. Launch Xamarin Workbook application.
2. Navigate to 'Tools' menu item present in menu bar and Expand it
3. Navigate to 'Preferences' menu item and select it
4. Navigate to 'update' tab item and select it
5. Navigate to 'Automatic check for updates' Combo box and Observe Inspect property

### Expected Behavior
Name property should be defined properly for Combo box (For e.g. 'Automatic check for updates')

### Actual Behavior
Name property is not defined for Combo-box
**Narrator Behavior:** Narrator is not announcing  name property for Combo box and only announcing list item selected in Combo box as  'Hourly, combo box, collapsed '
**NVDA Behavior:** NVDA is not announcing  name property for Combo box and only announcing list item selected in Combo box as  'Combo box,Hourly,  collapsed '

### User Impact
Users who rely on screen reader will not be able to know the Name of combo box

### Screenshots

![automatic_updates_combobox_name_property_narrator](https://user-images.githubusercontent.com/22956130/40723508-aefc536c-643c-11e8-8871-297a4ada38ba.png)
![automatic_updates_combobox_name_property_nvda](https://user-images.githubusercontent.com/22956130/40723515-b2ab4c3e-643c-11e8-9cb6-08b4a3f3f3d7.png)

![automatic_updates_combobox_name_property_inspect](https://user-images.githubusercontent.com/22956130/40723498-ab730a42-643c-11e8-9fbd-3d7272005791.png)

### Environment

Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
